src/components/MovieCard.vue

```diff
@@ -1,14 +1,21 @@
 <template>
   <div class="movie-card">
     <h3 class="movie-card__title">{{ movie.title }}</h3>
-    <img :src="movie.image" class="movie-card__image" :alt="imageAlt" />
-    <p class="movie-card__description">
-      {{ movie.description }}
+    <p class="movie-card__length">{{ movieLength(movie.length) }}</p>
```

Comment:
I would rethink naming here, because movieLength of movie.length is a bit weird 😄